src/query/buildRQL.js

```diff
@@ -96,9 +98,21 @@ var buildRQL = {
       if (isDeprecatedCallWithArgCountGreaterThan(queryBuilder, 2)) {
         // TODO: Delete legacy support, (Component, variables, rql) => rql`...`.
         node = queryBuilder(Component, variables, RelayQL);
-      } else {
+      } else if (isDeprecatedCallWithArgCountGreaterThan(queryBuilder, 0)) {
         node = queryBuilder(Component, variables);
       }
+      else {
+        var query = queryBuilder();
+        var rootCall = query.calls[0];
+        node = new GraphQL.Query(
+          rootCall.name,
+          rootCall.value,
+          query.fields,
+          [fragment],
+          query.metadata,
+          query.name
+        );
```

Comment:
This looks good. FYI the plan is to replace the `GraphQL.*` classes with plain objects and we try to avoid using `new GraphQL.*`, but there's no alternative here.